src/lib/graph/node.ts

```diff
@@ -1,7 +1,9 @@
 export type NodeState = '' | 'dirty' | 'in-progress' | 'pending' | 'done';
 
 export const STATE_DIRTY: NodeState = 'dirty';
-export const STATE_IN_PROGESS: NodeState = 'in-progress';
+export const STATE_IN_PROGRESS: NodeState = 'in-progress';
+/** @deprecated use STATE_IN_PROGRESS */
```

Comment:
This is private, hence you can remove it.

Reply:
Done.